packages/mask/src/plugins/CryptoartAI/Worker/index.ts

```diff
@@ -0,0 +1,8 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { base } from '../base'
+
+const worker: Plugin.Worker.Definition = {
+    ...base,
+    init(signal) {},
```

Comment:
why empty?

plz add comment into the method, thanks

Reply:
Just reference to other plugins such as https://github.com/DimensionDev/Maskbook/blob/develop/packages/mask/src/plugins/Collectible/Worker/index.ts#L5